sway/commands/move.c

```diff
@@ -618,7 +618,7 @@ static struct cmd_results *cmd_move_container(bool no_auto_back_and_forth,
 	if (root->fullscreen_global) {
 		arrange_root();
 	} else {
-		if (old_ws && !old_ws->node.destroying) {
+		if (old_ws) {
```

Comment:
Hm, `workspace_consider_destroy()` might've destroyed the workspace… Should we make that function return a bool to indicate whether the workspace ended up being destroyed, and reset `old_ws` to NULL in that case?